templates/fine-tune-GPTJ/README.md

```diff
@@ -0,0 +1,62 @@
+# Fine-Tuning LLMs on Anyscale with DeepSpeed
```

Comment:
Can we include this intro information as a table, including: how long this will take, a sample output, compute requirements, etc.

Basically this stuff:

<img width="1092" alt="Screen Shot 2023-07-09 at 11 52 21 PM" src="https://github.com/anyscale/templates/assets/3887863/0ee78bfc-075e-44cd-91de-d37e315d2036">

Example to copy from: https://raw.githubusercontent.com/ray-project/ray/master/doc/source/templates/03_serving_stable_diffusion/README.md